### ISSUE TITLE
Speed run adjustments

### ISSUE THREAD

**Issue by bizquig** [standalone/stretchwords#34]
- We need an easier way to get into the speed run. I have a tough time starting a game for some reason. Maybe a wholly different button to press on the menu page instead of pushing some place vague next to the "Start New Game" lettering.
- Along the same lines, once the speed run game is done there should be a query as to whether or not you want to play again instead of having to go to the homepage to start it again.
- Obviously, the highest score in this should be kept in stats.

**Comment by bengottlieb** [comment]
1: I figured this would come with a new 'landing page' re-design. I can make the button a little bigger for now.

2: I'll add this

3: This is already done, I just need to add a way to view your stats. #35

**Comment by bengottlieb** [comment]
Button embiggened. I'm going to add a new ticket for #2, I think it needs it's own thread.
